src/dscontrib/sustainability/utils.py

```diff
@@ -0,0 +1,58 @@
+import pandas as pd
+import pyspark.sql.functions as F
+
+
+def bootstrap(df, iterations, reporting_freq=10, strata=[],
```

Comment:
Documentation for this function would be super useful. A description of the parameters seems like the most critical. E.g. is `df` a pandas.DataFrame or a pyspark.DataFrame?